### PR TITLE
T5 269 add loadable

### PR DESCRIPTION
**Pull request by naeluh**
https://thirdandgrove.atlassian.net/browse/T5-269
- T5-269 Add @loadable/component to pages directory
- T5-269 Add loadable package

### CODE REVIEW

File: src/pages/acquia-engage-2020.js [hunk 1]
```diff
@@ -3,15 +3,23 @@ import PropTypes from 'prop-types';
...
+const LogoGrid = loadable(() => import('../components/LogoGrid'));
+const FullWidthSection = loadable(() =>
+  import('../components/FullWidthSection')
+);
+const SplitSection = loadable(() => import('../components/SplitSection'));
+const Layout = loadable(() => import('../components/layout'));
+const Button = loadable(() => import('../components/Button'));
+const { GetInTouch, SeeInsights } = loadable(() =>
+  import('../components/Prefooter')
+);
```
Comment (thepractice) [inline src/pages/acquia-engage-2020.js:22]: This is different from the others because it uses destructuring. So maybe this is causing the build to fail.

Comment (naeluh) [reply src/pages/acquia-engage-2020.js]: @thepractice thanks!

I figured it out we can do this 

```js
const LoadableGetInTouch = loadable(async () => {
  const { GetInTouch } = await import('../components/Prefooter');
  return GetInTouch;
});

const LoadableSeeInsights = loadable(async () => {
  const { SeeInsights } = await import('../components/Prefooter');
  return SeeInsights;
});
```

**Review by CoderBrandon — APPROVED** [review]

File: src/pages/acquia-engage-2020.js [hunk 2]
```diff
@@ -3,15 +3,24 @@ import PropTypes from 'prop-types';
...
 import { container, mediaQueries, weights, colors, fonts } from '../styles';
-import Button from '../components/Button';
-import { GetInTouch, SeeInsights } from '../components/Prefooter';
+
+const NewsletterSimpleOverlay = loadable(() =>
+  import('../components/NewsletterForm/NewsletterSimpleOverlay')
+);
+const GetInTouch = loadable(() => import('../components/Prefooter/GetInTouch'));
+const SeeInsights = loadable(() =>
+  import('../components/Prefooter/SeeInsights')
```
Comment (jds-tag) [inline src/pages/acquia-engage-2020.js:15]: Should this be on the same line? Only asking bc this is the first line break I'm noticing.

File: src/pages/acquia-engage-2020.js [hunk 3]
```diff
@@ -3,15 +3,24 @@ import PropTypes from 'prop-types';
...
+);
+const LogoGrid = loadable(() => import('../components/LogoGrid'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/acquia-engage-2020.js:18]: Same line?

File: src/pages/boston.js [hunk 4]
```diff
@@ -21,7 +16,17 @@ import {
   contValues,
   pLight,
 } from '../styles';
-import Button from '../components/Button';
+
+const Layout = loadable(() => import('../components/layout'));
+const ImageSlider = loadable(() => import('../components/ImageSlider'));
+const Button = loadable(() => import('../components/Button'));
+const SplitSection = loadable(() => import('../components/SplitSection'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/boston.js:24]: Go ahead and keep this on two lines or move to same line depending on how you address feedback above/feel what makes the most sense for formatting.

File: src/pages/boston.js [hunk 5]
```diff
@@ -21,7 +16,17 @@ import {
...
+  import('../components/FullWidthSection')
+);
+const TextWrapper = loadable(() =>
```
Comment (jds-tag) [inline src/pages/boston.js:27]: Ditto same line vs. new line.

File: src/pages/careers.js [hunk 6]
```diff
@@ -2,10 +2,14 @@ import React from 'react';
...
 import styled from '@emotion/styled';
 import { css } from '@emotion/react';
+import loadable from '@loadable/component';
 
 import { fonts, weights, container, mediaQueries } from '../styles';
-import Layout from '../components/layout';
-import FullWidthSection from '../components/FullWidthSection';
+
+const Layout = loadable(() => import('../components/layout'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/careers.js:10]: Same line vs new line.

File: src/pages/drupal-support.js [hunk 7]
```diff
@@ -2,19 +2,23 @@ import React from 'react';
...
-import Layout from '../components/layout';
 import { container, mediaQueries, weights, colors } from '../styles';
-import ContactForm from '../components/ContactForm';
-import CTA from '../components/CTA';
-import CTAGrid from '../components/CTAGrid';
 import drupalLogo from '../images/drupal-support/drupal-logo.png';
 import preserver from '../images/drupal-support/preserver.png';
 
+const Layout = loadable(() => import('../components/layout'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/drupal-support.js:12]: Same line vs. new lines. I mean maybe this is just prettier. I am only commenting them all for convenience.

File: src/pages/drupalcon-2021-event.js [hunk 8]
```diff
@@ -3,16 +3,22 @@ import PropTypes from 'prop-types';
...
+const Button = loadable(() => import('../components/Button'));
+const Quote = loadable(() => import('../components/ContentBody/Quote'));
+const ButtonForm = loadable(() => import('../components/ButtonForm'));
+const BeUs = loadable(() => import('../components/Prefooter/BeUs'));
+const WhatWeDo = loadable(() => import('../components/Prefooter/WhatWeDo'));
+
+const Layout = loadable(() => import('../components/layout'));
+const InsightsSlider = loadable(() => import('../components/InsightsSlider'));
+const SplitSection = loadable(() => import('../components/SplitSection'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/drupalcon-2021-event.js:19]: Same line vs. new line.

File: src/pages/index.js [hunk 9]
```diff
@@ -1,59 +1,48 @@
...
-import LogoGrid from '../components/LogoGrid';
-import SplitSection from '../components/SplitSection';
-import { ContactUs, BeUs } from '../components/Prefooter';
-import { colors } from '../styles';
 import { useHasBeenVisible } from '../hooks/useVisibility';
-import FullWidthSection from '../components/FullWidthSection';
-import { NewsletterFullWidthSection } from '../components/NewsletterForm';
+import { colors } from '../styles';
+
+const NewsletterFullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/index.js:9]: Same line vs. new line

File: src/pages/index.js [hunk 10]
```diff
@@ -1,59 +1,48 @@
...
+  import('../components/NewsletterForm/NewsletterFullWidthSection')
+);
+const BeUs = loadable(() => import('../components/Prefooter/BeUs'));
+const ContactUs = loadable(() => import('../components/Prefooter/ContactUs'));
+
```
Comment (jds-tag) [inline src/pages/index.js:14]: Rm xtra line?

File: src/pages/index.js [hunk 11]
```diff
@@ -1,59 +1,48 @@
...
+const Layout = loadable(() => import('../components/layout'));
+const ProjectsSlider = loadable(() => import('../components/ProjectsSlider'));
+const InsightsSlider = loadable(() => import('../components/InsightsSlider'));
+const CapabilitiesSlider = loadable(() =>
```
Comment (jds-tag) [inline src/pages/index.js:18]: Same line vs. new line

File: src/pages/index.js [hunk 12]
```diff
@@ -1,59 +1,48 @@
...
+  import('../components/CapabilitiesSlider')
+);
+const LogoGrid = loadable(() => import('../components/LogoGrid'));
+const SplitSection = loadable(() => import('../components/SplitSection'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/index.js:23]: Same line vs. new line

File: src/pages/insights.js [hunk 13]
```diff
@@ -11,12 +12,15 @@ import {
...
   weights,
 } from '../styles';
-import ArticlePreview from '../components/ArticlePreview';
-import Button from '../components/Button';
-import Layout from '../components/layout';
-import FullWidthSection from '../components/FullWidthSection';
 import { ensureTrailingSlash } from '../util';
 
+const Layout = loadable(() => import('../components/layout'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/insights.js:18]: Same line vs. new line.

File: src/pages/partners/acquia.js [hunk 14]
```diff
@@ -15,12 +12,21 @@ import {
...
   pLight,
 } from '../../styles';
-import SplitSection from '../../components/SplitSection';
 import InsightsSlider from '../../components/InsightsSlider';
 import Quote from '../../components/ContentBody/Quote';
 import CTA from '../../components/CTA';
 import CTAGrid from '../../components/CTAGrid';
 
+const Layout = loadable(() => import('../../components/layout'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/partners/acquia.js:21]: Same line vs. new line.

File: src/pages/partners/acquia.js [hunk 15]
```diff
@@ -15,12 +12,21 @@ import {
...
+  import('../../components/FullWidthSection')
+);
+const SplitSection = loadable(() => import('../../components/SplitSection'));
+const LogoGrid = loadable(() => import('../../components/LogoGrid'));
+const ProjectsSlider = loadable(() =>
```
Comment (jds-tag) [inline src/pages/partners/acquia.js:26]: Same line vs. new line.

File: src/pages/partners/big-commerce.js [hunk 16]
```diff
@@ -1,18 +1,26 @@
...
 import { colors } from '../../styles';
-import SplitSection from '../../components/SplitSection';
-import InsightsSlider from '../../components/InsightsSlider';
-import Quote from '../../components/ContentBody/Quote';
 import { partnersProjects, list } from '../../styles/custom-css';
-import CTA from '../../components/CTA';
 import EliteSRC from '../../../static/images/Partner_Wordmark-Elite_1col.svg';
 
+const Layout = loadable(() => import('../../components/layout'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/partners/big-commerce.js:11]: Same line vs. new lines.

File: src/pages/partners/big-commerce.js [hunk 17]
```diff
@@ -1,18 +1,26 @@
...
+  import('../../components/FullWidthSection')
+);
+const SplitSection = loadable(() => import('../../components/SplitSection'));
+const Quote = loadable(() => import('../../components/ContentBody/Quote'));
+const CTA = loadable(() => import('../../components/CTA'));
+const InsightsSlider = loadable(() =>
```
Comment (jds-tag) [inline src/pages/partners/big-commerce.js:17]: Same line vs. new lines (next two blocks)

File: src/pages/partners/drupal.js [hunk 18]
```diff
@@ -15,10 +12,21 @@ import {
   container,
   pLight,
 } from '../../styles';
-import SplitSection from '../../components/SplitSection';
-import InsightsSlider from '../../components/InsightsSlider';
-import Quote from '../../components/ContentBody/Quote';
-import CTA from '../../components/CTA';
+
+const Layout = loadable(() => import('../../components/layout'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/partners/drupal.js:17]: Same line vs. new lines.

File: src/pages/partners/drupal.js [hunk 19]
```diff
@@ -15,10 +12,21 @@ import {
...
+  import('../../components/FullWidthSection')
+);
+const SplitSection = loadable(() => import('../../components/SplitSection'));
+const LogoGrid = loadable(() => import('../../components/LogoGrid'));
+const Quote = loadable(() => import('../../components/ContentBody/Quote'));
+const CTA = loadable(() => import('../../components/CTA'));
+const InsightsSlider = loadable(() =>
```
Comment (jds-tag) [inline src/pages/partners/drupal.js:24]: Same line vs. new lines (next two blocks).

File: src/pages/partners/gatsby.js [hunk 20]
```diff
@@ -1,13 +1,19 @@
...
-import FullWidthSection from '../../components/FullWidthSection';
 import { colors, mediaQueries, fonts, weights, container } from '../../styles';
-import InsightsSlider from '../../components/InsightsSlider';
-import Quote from '../../components/ContentBody/Quote';
-import CTA from '../../components/CTA';
+
+const Quote = loadable(() => import('../../components/ContentBody/Quote'));
+const CTA = loadable(() => import('../../components/CTA'));
+const Layout = loadable(() => import('../../components/layout'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/partners/gatsby.js:11]: Same line vs. new lines (next two blocks)

File: src/pages/partners/shopify.js [hunk 21]
```diff
@@ -1,18 +1,24 @@
...
-import InsightsSlider from '../../components/InsightsSlider';
-import CTAGrid from '../../components/CTAGrid';
-import Capability from '../../components/Capability';
-import Improvement from '../../components/Improvement';
-import ContactForm from '../../components/ContactForm';
 import { colors, mediaQueries } from '../../styles';
 
+const Layout = loadable(() => import('../../components/layout'));
+const Button = loadable(() => import('../../components/Button'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/partners/shopify.js:10]: Same line vs. new line.

File: src/pages/partners/shopify.js [hunk 22]
```diff
@@ -1,18 +1,24 @@
...
+  import('../../components/FullWidthSection')
+);
+const LogoGrid = loadable(() => import('../../components/LogoGrid'));
+const InsightsSlider = loadable(() =>
```
Comment (jds-tag) [inline src/pages/partners/shopify.js:14]: Same line vs. new lines.

File: src/pages/san-francisco.js [hunk 23]
```diff
@@ -21,7 +16,17 @@ import {
   contValues,
   pLight,
 } from '../styles';
-import Button from '../components/Button';
+
+const Layout = loadable(() => import('../components/layout'));
+const ImageSlider = loadable(() => import('../components/ImageSlider'));
+const Button = loadable(() => import('../components/Button'));
+const SplitSection = loadable(() => import('../components/SplitSection'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/san-francisco.js:24]: Same line vs. new lines (next two blocks)

File: src/pages/work.js [hunk 24]
```diff
@@ -4,16 +4,20 @@ import { Spring } from 'react-spring/renderprops';
...
 import { useHasBeenVisible } from '../hooks/useVisibility';
-import VideoSection from '../components/VideoSection';
 import { ensureTrailingSlash } from '../util';
-import CTA from '../components/CTA';
+
+const Layout = loadable(() => import('../components/layout'));
+const CTA = loadable(() => import('../components/CTA'));
+const VideoSection = loadable(() => import('../components/VideoSection'));
+const FakeButton = loadable(() => import('../components/FakeButton'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/pages/work.js:18]: Same line vs. new lines.

File: src/pages/work.js [hunk 25]
```diff
@@ -70,15 +74,18 @@ const Project = ({ study, index }) => {
...
                     margin-bottom: 0;
 
                     > div {
                       padding-bottom: ${index % 2 ? '76% !important' : '100%'};
-                      padding-bottom: ${index % 4 === 2
-                        ? '131% !important'
-                        : '100%'};
+                      padding-bottom: ${
+                        index % 4 === 2 ? '131% !important' : '100%'
+                      }import { loadable } from '@loadable/component';
```
Comment (jds-tag) [inline src/pages/work.js:87]: Is this import statement supposed to be here?

File: src/templates/insights.js [hunk 26]
```diff
@@ -3,15 +3,19 @@ import PropTypes from 'prop-types';
...
-import InsightsSlider from '../components/InsightsSlider';
-import {
-  NewsletterFullWidthSection,
-  NewsletterOverlay,
-} from '../components/NewsletterForm';
+
+const Layout = loadable(() => import('../components/layout'));
+const ContentBody = loadable(() => import('../components/ContentBody'));
+const InsightsSlider = loadable(() => import('../components/InsightsSlider'));
+const NewsletterFullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/templates/insights.js:13]: Same line vs. new lines (next two blocks)

File: src/templates/job.js [hunk 27]
```diff
@@ -1,11 +1,15 @@
...
 import { css } from '@emotion/react';
+import loadable from '@loadable/component';
 
 import { mediaQueries, container, fonts, weights } from '../styles';
-import Layout from '../components/layout';
-import FullWidthSection from '../components/FullWidthSection';
-import Button from '../components/Button';
+
+const Layout = loadable(() => import('../components/layout'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/templates/job.js:9]: Same line vs. new lines.

File: src/templates/legacyInsights.js [hunk 28]
```diff
@@ -1,10 +1,14 @@
...
 import PropTypes from 'prop-types';
 import { css } from '@emotion/react';
+import loadable from '@loadable/component';
 
 import { fonts, mediaQueries } from '../styles';
-import Layout from '../components/layout';
-import FullWidthSection from '../components/FullWidthSection';
+
+const Layout = loadable(() => import('../components/layout'));
+const FullWidthSection = loadable(() =>
```
Comment (jds-tag) [inline src/templates/legacyInsights.js:9]: Same line vs. new lines.

**Review by jds-tag — COMMENTED** [review]
Just the same formatting question in many places (could just be prettier) and one question. Solid though!!!

File: src/templates/legacyInsights.js (hunk 28 above)
Comment (naeluh) [reply src/templates/legacyInsights.js]: @jds-tag Yes I added a commit that turn the print width to 120 I think it works better. thanks!

**Review by jds-tag — APPROVED** [review]
LGTM! This is a big diff so please make sure this gets regression tested/goes through QA. Thanks!

**Pull request closed without merging**